lib/adal.js

```diff
@@ -546,6 +546,11 @@ var AuthenticationContext = (function () {
         }
 
         var urlNavigate = this.instance + tenant + '/oauth2/logout?' + logout;
+
+        if (this.config.logOutUri) {
+            urlNavigate = this.instance + this.config.logOutUri;
+        }
+
         this.info('Logout navigate to: ' + urlNavigate);
```

Comment:
@NickAb I think logOutUri should be an absolute uri like redirectUri. Users configuring this already knows about the instance. We should use it as is. Also, can we structure this a little differently like this:

        var urlNavigate; 
        if (this.config.logOutUri) {
             urlNavigate = this.config.logOutUri;
         }
         else {
              if (this.config.postLogoutRedirectUri) {
                  logout = 'post_logout_redirect_uri=' + encodeURIComponent(this.config.postLogoutRedirectUri);
              }
             urlNavigate = this.instance + tenant + '/oauth2/logout?' + logout;
         }